airflow/hooks/dbapi.py

```diff
@@ -178,7 +179,25 @@ def get_first(self, sql, parameters=None):
                     cur.execute(sql)
                 return cur.fetchone()
 
-    def run(self, sql, autocommit=False, parameters=None, handler=None):
+    @staticmethod
+    def maybe_split_sql_string(sql: str) -> List[str]:
```

Comment:
We want to keep Airflow 2.2 compatible for all providers, so you need to add some code to make this change compatible with older versions of Airflow as well. Most often it is done by detecting the function and if a given version of Airflow does not have a function, then using the code that was explicitly copied to the provider's package.

Reply:
Thanks. Continue working on this PR